### PR TITLE
[RF-5686] Add multibrowser, multiscript support. Bring some stuff up to date, 🔥 some other stuff

### PR DESCRIPTION
**Pull request by magni-**
Sorry about the huge diff - might be easier to go commit by commit.

### CODE REVIEW

File: bin/rainforest_test [hunk 1]
```diff
@@ -3,6 +3,31 @@
...
 
-RainforestRubyRuntime::Runner.new(browsers: %w(firefox)).run ARGF.read
+options = {browsers: %i(chrome)}
+OptionParser.new do |opts|
+  opts.banner = "Usage: ruby rainforest_test [options] [file_paths]"
 
-puts "Your test completed successfuly!"
+  opts.on('--browsers X,Y,Z', Array, 'Browsers') do |browsers|
+    invalid_browsers = browsers - RainforestRubyRuntime::Runner::BROWSERS
+    puts invalid_browsers
```
Comment (grodowski) [inline bin/rainforest_test]: Is that puts needed if we raise anyway in L13?

File: Rakefile [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-Rake::TestTask.new do |t|
-  t.pattern = "test/**/*_test.rb"
+begin
+  require 'rspec/core/rake_task'
+  ENV['CAPYBARA_DRIVER'] = 'selenium'
+  RSpec::Core::RakeTask.new(:spec) do |t|
+    t.rspec_opts = '--color --format documentation'
+  end
+rescue LoadError
+  # No RSpec
```
Comment (grodowski) [inline Rakefile]: When can this happen? Can we output something useful here?

File: lib/rainforest_ruby_runtime/version.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 module RainforestRubyRuntime
-  VERSION = "0.0.3"
+  VERSION = "0.1.0"
```
[the remark this answers is not in the document]
Comment (magni-) [reply lib/rainforest_ruby_runtime/version.rb]: 🤷‍♂️

File: lib/rainforest_ruby_runtime/drivers/sauce.rb [hunk 4]
```diff
@@ -7,24 +7,35 @@ def initialize(options)
...
+          end
         end
       end
 
       private
 
+      def apply_config
+        ::Sauce.config do |c|
+          c[:browsers] = browsers
+          c[:sauce_connect_4_executable] = 'vendor/sauce/sc'
```
[the remark this answers is not in the document]
Comment (magni-) [reply lib/rainforest_ruby_runtime/drivers/sauce.rb]: I'll remove and see if it still works 🤞

[the remark this answers is not in the document]
Comment (magni-) [reply lib/rainforest_ruby_runtime/drivers/sauce.rb]:
```
CAPYBARA_DRIVER=sauce ruby bin/rainforest_test --browsers chrome testing-test.rb
Running against chrome

Rainforest
Please install the `sauce-connect` gem if you intend on using Sauce Connect with your tests!

Finished in 0.04494 seconds (files took 2.68 seconds to load)
0 examples, 0 failures
```

Confusingly enough, it's required

[the remark this answers is not in the document]
Comment (grodowski) [reply lib/rainforest_ruby_runtime/drivers/sauce.rb]: I succeeded in setting up `--prepare-sauce` (with github fake url) on my ubuntu vm, but this line `vendor/sauce/sc` wasn't working until while executing tests. I changed it to `File.join(RainforestRubyRuntime.root, '/vendor/sauce/sc')` where `root` is 
```
module RainforestRubyRuntime
  def root
     File.dirname(__dir__)
  end
end
```

**Pull request merged**